src/elements/content-sidebar/activity-feed/common/activity-message/ToggleMoreLessButton.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @flow
+ * @file Show more/less button
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import PlainButton from '../../../../../components/plain-button';
+
+import messages from './messages';
+
+type Props = {
+    isMore: boolean,
+    onClick: Function,
+};
+
+const ToggleMoreLessButton = ({ isMore, onClick }: Props): React.Node => (
+    <PlainButton type="button" className="bcs-ActivityMessage-toggleMoreLess" onClick={onClick}>
```

Comment:
Minor, but can we move `type` to the end?

Reply:
Can we have lint rule for this?

Reply:
Sounds fantastic. The issue we've had in the past with new rules like that is that there's lots of non-conforming code that needs to be updated.